gridsome.client.js

```diff
@@ -4,19 +4,42 @@ export default function (Vue, options, context) {
         return;
     }
 
+    if (options.excludeRoutes && typeof options.excludeRoutes !== 'string' && !Array.isArray(options.excludeRoutes)) {
```

Comment:
I used an 'and' because of the negation. The error should fire "if the option is a truthy-value and the option is not a string and not an array", as soon as it is a string or an array, the statement will evaluate to `false` and the error won’t be thrown.

I think the way you’re suggesting it would fire the error even if the option was a string or an array, because the nested clause would evaluate to `true`, since a string won’t be an array and an array won’t be a string.

In any case, I agree that having too many options here doesn’t make sense, so I’d say we just allow an array of strings.